src/utils/prefix.js

```diff
@@ -0,0 +1,13 @@
+const { getInput } = require("./action");
+
+/**
+ * Returns the prefix to use based on settings.
+ * @param {string} linter - the name of the linter.
+ * @returns {string} - the prefix to add to the command.
+ */
+function prefix(linter) {
+	const inputPrefix = getInput(`${linter}_command_prefix`) || "";
+	return inputPrefix ? `${inputPrefix} ` : "";
```

Comment:
You've added defaults for all options in `action.yml`, so this code can be simplified:

```suggestion
	return getInput(`${linter}_command_prefix`, true);
```